lib/bundler/cli/outdated.rb

```diff
@@ -59,18 +57,15 @@ def run
         definition_resolution.call
       end
 
-      Bundler.ui.info ""
+      Bundler.ui.info("")
 
       # Loop through the current specs
       gemfile_specs, dependency_specs = current_specs.partition do |spec|
-        current_dependencies.key? spec.name
+        current_dependencies.key?(spec.name)
```

Comment:
Let's keep the old style of parentheses everywhery, and focus on the refactoring.

Reply:
not agree, I my opinion the parentheses should be mandatory, this case with one parameter there is no confusion or doubt but when people add more parameters and logic in the parameters can be confused.